### PR TITLE
STEM: Eligibility Fixes

### PR DESCRIPTION
**Pull request by zurbergram**
## Description
https://github.com/department-of-veterans-affairs/va.gov-team/issues/6285
https://github.com/department-of-veterans-affairs/va.gov-team/issues/6284
https://github.com/department-of-veterans-affairs/va.gov-team/issues/6280
https://github.com/department-of-veterans-affairs/va.gov-team/issues/6290

## Testing done
- local testing

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]

File: src/applications/edu-benefits/1995/containers/StemEligibilityView.jsx [hunk 1]
```diff
@@ -70,8 +70,13 @@ export class StemEligibilityView extends React.Component {
 
     const isEnrolledStemCheck = isEnrolledStem || isPursuingTeachingCert;
 
-    const isEnrolledStemText =
-      'Are enrolled in a STEM undergraduate degree program, or have earned a STEM degree and are now pursuing a teaching certification';
+    const isEnrolledStemText = (
+      <span>
+        Are enrolled in a STEM undergraduate degree program, <strong>or</strong>
+        &nbsp;have earned a STEM degree and are now pursuing a teaching
+        certification'
```
Comment (bkjohnson) [inline src/applications/edu-benefits/1995/containers/StemEligibilityView.jsx]: Is this trailing single quote supposed to be there?

**Review by bkjohnson — APPROVED** [review]
I might have found a content issue, but keep in mind that a platform review isn't looking for those.  Things like that should be caught in peer review.

Also, the unit test coverage could be a bit better.

File: src/applications/edu-benefits/1995/containers/StemEligibilityView.jsx (hunk 1 above)
Comment (zurbergram) [reply src/applications/edu-benefits/1995/containers/StemEligibilityView.jsx]: no, removed

**Pull request merged**